lib/mix/lib/mix/dep/lock.ex

```diff
@@ -17,7 +17,7 @@ defmodule Mix.Dep.Lock do
     with {:ok, contents} <- File.read(lockfile),
          assert_no_merge_conflicts_in_lockfile(lockfile, contents),
          {:ok, quoted} <- Code.string_to_quoted(contents, opts),
-         {%{} = lock, _binding} <- Code.eval_quoted(quoted, opts) do
+         {%{} = lock, _binding} <- Code.eval_quoted(quoted) do
```

Comment:
We need to pass the `file` to eval_quoted so it is used in error reports in the odd chance there is a syntax error in the lock file (for example, if someone changes it manually). We could have two separate `opts`, but sharing them is also fine as is. Thank you!

Reply:
Should it be then?

```suggestion
         {%{} = lock, _binding} <- Code.eval_quoted(quoted, [], opts) do
```

Reply:
Oh, I see, thank you! @irisTa56, can you please apply the suggestion above?

Reply:
Thank you!
I made a new commit before noticing the suggestion, sorry.